### PR TITLE
Added release announce for scalatra 2.6.5

### PR DESCRIPTION
**Pull request by magnolia-k**
There was an issue saying that project news has not been updated, so I made an announcement for the latest release.

https://github.com/scalatra/scalatra/issues/938

### CODE REVIEW

**Comment by takezoe** [comment]
Thanks. To be honest, I didn't make announcements for minor updates because it was a bother, just updated the versions table.

File: content/news/2019-02-15-scalatra-2-6-5-released.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Scalatra 2.6.5 is out"
+layout: news
+author: Naoki Takezoe
+twitter: takezoen
```
Comment (takezoe) [inline content/news/2019-02-15-scalatra-2-6-5-released.md]: Why aren't `author` and `twitter` yours? :-)

File: content/news/2019-02-15-scalatra-2-6-5-released.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+date: 2019-02-15
+---
+
+The Scalatra team is pleased to announce the release of version 2.6.5 of the framework. 
+
+<!--more-->
+
+Here's the full list of changes from version 2.6.4:
```
Comment (takezoe) [inline content/news/2019-02-15-scalatra-2-6-5-released.md]:
```suggestion
Here's the full list of changes from version 2.6.5:
```

File: content/news/2019-02-15-scalatra-2-6-5-released.md (hunk 1 above)
Comment (magnolia-k) [reply content/news/2019-02-15-scalatra-2-6-5-released.md]: oh! I'll fix it

File: content/news/2019-02-15-scalatra-2-6-5-released.md (hunk 2 above)
Comment (magnolia-k) [reply content/news/2019-02-15-scalatra-2-6-5-released.md]: fix it

**Comment by magnolia-k** [comment]
I think that frequent announcements are unnecessary, but because it is not good that they are not considered to be maintained, it may be sufficient to indicate the fact that they are being maintained :)

**Pull request merged**